src/main/kotlin/com/hypto/iam/server/service/OrganizationsService.kt

```diff
@@ -44,15 +48,26 @@ class OrganizationsServiceImpl : KoinComponent, OrganizationsService {
     override suspend fun createOrganization(
         name: String,
         description: String,
-        adminUser: AdminUser
+        adminUser: AdminUser,
+        passcodeStr: String?
     ): Pair<Organization, Credential> {
-        val organizationId = idGenerator.organizationId()
+        val passcode = passcodeStr?.let {
```

Comment:
CreateOrganization should support org creation based on either
1. Email verification.
2. Based on the secret x-api-key. (For support and admin teams to create org on behalf of customer)

This code handles `#1` but `#2` is not handled. Can you please update to support org creation if valid x-api-key is sent from client ?